**Quantitative and computational biology**

GSK Graduate School

Spring 2016

Course Syllabus

Saturday, February 6, 2016 – Monday, May 23, 2016

The introductory “bootcamp” (Saturday, February 6, 2016, 9:00 AM – 5:00
PM) + eight sessions on Mondays 1:00 PM – 5:00 PM (February 8, February
22, March 7, March 21, April 11, April 26, May 9, and May 23, 2016)

Location: Zuckerman Research Center

ZRC 138

417 East 68th Street\
New York, NY 10065

Course Instructors: Sonya Hanson
([*sonya.hanson@choderalab.org*](mailto:sonya.hanson@choderalab.org)),
Hatice Ulku Osmanbeyoglu
(osmanbeh[*@mkscc.com*](mailto:uosmanbey@gmail.com)), Yuri Pritykin
([*pritykin@cbio.mskcc.org*](mailto:pritykin@cbio.mskcc.org)), Lee
Zamparo ([*zamparol@cbio.mskcc.org*](mailto:zamparol@cbio.mskcc.org)).

This course will not be graded.

**Summary **

Students in this course will learn to apply quantitative exploratory
data analysis techniques to different forms of experimental data. The
course will begin with a software skills “bootcamp”, which will
introduce students to computing via the UNIX shell, and to computing in
the R programming language. The remaining lessons will be a blend of
practical skills and theoretical concepts. Students will become
comfortable performing exploratory data analysis, and will understand
how concepts from statistics underlie the tools they use. These skills
will enable them both to collaborate effectively with computational
biologists, as well as to carry out their own computational experiments.

Specific topics include: practical aspects of data formatting and
management; visualization of data and model parameters; an introduction
to probability, elementary statistics and hypothesis testing;
experimental design and tools for differential expression analysis in
RNA-seq and ChIP-seq; interpreting model fit, correct interpretation of
statistical evidence, and multiple hypothesis testing correction
methods; common data-structures for working with biological sequences;
enrichment testing for ranked gene sets; common bioinformatics tools and
data quality assessment tools; introduction to structural biology and
tools for visualization.

Towards the end of the course, students are encouraged to do a project
on bioinformatic data analysis and present their results in the last
class.

**Class Sessions**

0.  The bootcamp: Saturday, February 6, 2016.

Basic tools for research computing. Installation of shell, git, R and
Rstudio. Introduction to shell computing, the R programming language,
basic statistics and visualization.

0.  Lab 1: Monday, February 8, 2016

Introduction to statistics and probability. Review of loading and
plotting data in R.

0.  Lab 2: Monday, February 22, 2016

Review of basic stats, mean and standard deviation, normal distribution,
other types of distributions, intro to probability, hypothesis testing,
review of plotting techniques.

0.  Lab 3: Monday, March 7, 2016

Practical RNA-seq, a tour of DE-seq.

0.  Lab 4: Monday, March 21, 2016

Practical ChIP-seq, a pipeline approach. Experimental design. Mapping
software. Peak calling, and non-peak based analysis. Analysis of
enriched areas. Viewing ChIP-seq data in genome browsers.

0.  Lab 5: Monday, April 11, 2016

Interpreting P-values and Multiple Hypothesis Testing. Exploratory data
analysis.

0.  Lab 6: Monday, April 26, 2016

Genomic data-structures in R and Bioconductor: GenomicRanges,
GenomicFeatures, BSgenome, AnnotationDbi. Project consultation.

0.  Lab 7: Monday, May 9, 2016

Bioinformatics tools: browsers, GO enrichment, gene set enrichment
analysis. Project consultation.

0.  Lab 8: Monday, May 23, 2016

More examples of statistical data analysis. Project presentations.

**Books and Materials **

Students will need a laptop computer on which they can install software
(R and R Studio), and bring to class. All software used in the course is
free, and run on recent versions of Linux, Mac OS X, and Microsoft
Windows.

R: [*http://www.r-project.org*](http://www.r-project.org)

Rstudio: [*http://www.rstudio.com*](http://www.rstudio.com/)

Gitshell (Windows only):
[*https://git-for-windows.github.io/*](https://git-for-windows.github.io/)

The course does not require the use of a specific textbook, but instead
makes use of the following online resources:

\[Software Capentry\]

-   Introduction to the Shell:
    > [*http://swcarpentry.github.io/shell-novice/*](http://swcarpentry.github.io/shell-novice/)

-   R for Reproducible Scientific Analysis:
    > [*http://swcarpentry.github.io/r-novice-gapminder/*](http://swcarpentry.github.io/r-novice-gapminder/)

\[qBio-I course materials (courtesy of Jason Banfelder)\]

-   [*http://physiology.med.cornell.edu/people/banfelder/qbio/schedule\_2015/\#qbio-i*](http://physiology.med.cornell.edu/people/banfelder/qbio/schedule_2015/#qbio-i)

\[Biomedical Data Science (courtesy of Raphael Irizarry and Michael
Love)\]

-   [*http://genomicsclass.github.io/book/*](http://genomicsclass.github.io/book/)

\[qBio-GSK 2016 Course Repository\]:

-   [*https://github.com/qbio-gsk/qbio-gsk*](https://github.com/qbio-gsk/qbio-gsk)

-   Other resources are linked from lesson plans in the repository

**Assessment**

This class will not be graded. However, periodic student surveys in
class and online are planned to assess if the presented material is
properly learnt. All students will also be asked to complete a survey at
the end of the course, to obtain feedback on the course and assess if
this experimental format is successful and what changes need to be done
when the course is offered in future years.
